crates/metal-mir/src/types/visibility.rs

```diff
@@ -0,0 +1,6 @@
+#[derive(Debug, Clone, Copy)]
+pub enum Visibility {
+    Public,
+    ParcelLevel,
+    Private,
```

Comment:
There isn't such thing as "private" (fully private items would be quite literally unusable), but there is "local" visibility (the default one, if `pub`/`pub(path)` is not specified) that makes the item visible only to other items in the current module (and maybe to items of children modules, can't remember for sure)

Reply:
Private is just more of a generalization. Most people call them private not local functions from my experience at least so I just named it Private. I don't think it needs changing, but if you want to feel free